CI_YAML.md

```diff
@@ -69,6 +69,23 @@ targets:
        ["framework", "hostonly"]
    postsubmit_properties:
      - upload_metrics: "true"
+
+#
+# Devicelab example:
+# For tests that are located https://github.com/flutter/flutter/tree/master/dev/devicelab/bin/tasks:
+# 1) a `taskname` property is required, which should match the task name
+# 2) target name follows format of `<platform> <taskname>`
+# 3) update `tags` accordingly
+#
+# Here is the target config for a task named: `analyzer_benchmark.dart`.
+ - name: Linux_android analyzer_benchmark
+   recipe: devicelab/devicelab_drone
```

Comment:
What does these actually mean? (e.g. presubmit: false --- I assume it means don't run on presubmit, but I'm guessing --- and "timeout" -- from @CaseyHillers comment I'm guessing it's in minutes, but again I don't know... And if there is none (as suggested) what does that mean?

Reply:
Added a comment to `targets` section to explain recipes.

Removing the `timeout` to avoid confusion. We have default timeout setup from recipes side to control test timeout if hanging.